src/pages/solution/adobe-analytics/migrate-to-edge-network.md

```diff
@@ -46,7 +46,6 @@ Other foundational extensions include the [Consent for Edge Network extension](.
 | 6. **Remove the Analytics extension** dependency and extension registration from your mobile app code. <br/> **NOTE:** You should still keep the Analytics extension installed in the mobile property (tag) to ensure published versions of your app (pre-migration) continue to work seamlessly. |  ✅ | ✅ |
 | 7 .**Use Edge.sendEvent API** to send data in XDM format to Edge Network based on the schema you have defined. |  ✅ |  |
 | 8. **Keep existing MobileCore.trackAction / MobileCore.trackState API calls** to send data in context data format to Experience Platform.|  | ✅ |
-| 9. **Map your context data to XDM** in Data Prep for Data Collection.|  | ✅ |
```

Comment:
can we keep this step as an optional step if needed for other services?

9. **(Optional) Map your context data to XDM** in Data Prep for Data Collection if required for other services. Not required for Analytics. |